breathecode/mentorship/tests/urls/tests_academy_mentor_id.py

```diff
@@ -519,14 +551,20 @@ def test__post__with_one_mentor_profile__changing_to_a_success_status(self):
     def test__post__with_one_mentor_profile__changing_to_a_success_status__raise_a_exception(self):
         statuses = ['INVITED', 'ACTIVE', 'UNLISTED', 'INNACTIVE']
         valid_statuses = ['ACTIVE', 'UNLISTED']
+        profile_academy = {
+            'first_name': self.bc.fake.name(),
+            'last_name': self.bc.fake.name(),
+            'phone': self.bc.fake.phone_number(),
```

Comment:
Add a case when the mentor is being changed and the profile academy doesn't have its property set